### PR TITLE
 Cloudevent specifications added in features

### LINKED ISSUE #443: [Feature] [Configuration Card][Operator Users]  CloudEvents specification
#### Current Behavior

https://meshery.io/features --> Configuration Card--> does not mention using CloudEvents

#### Desired Behaviour

https://meshery.io/features 
Use Existing card: [Feature] [Configuration] or a new card  [Feature][Configuration] [Operator Users]
Add  items: Management of task/operation lifecycle for Operators Actors using CloudEvents specification
Beautify/Rephrase "Operatore with best practices"
#### Mockups

---
**[Contributor Guide](https://docs.meshery.io/project/contributing)**

### PR DESCRIPTION
**Pull request by soham4abc**
** cloudevent specifications added in features**

This PR fixes #443 

**Notes for Reviewers**
added cloudevent specifications in features and did some code formatting

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: collections/_pages/features.html [hunk 1]
```diff
@@ -16,30 +16,32 @@
...
+      - text:
+        highlight:
     call_to_action:
       link: /#getting-started
       text: Get started
   - name: Configuration
     color: "#00B39F"
     features:
-      - text: Operatore with best practices
+      - text: Operator with optimum practices
```
Comment (leecalcote) [inline collections/_pages/features.html:29]:
```suggestion
      - text: Operate with confidence
```

File: collections/_pages/features.html [hunk 2]
```diff
@@ -81,32 +83,38 @@
...
-			</li>
-			{% for item in feature.features %}
-				<li {% if item.highlight %} class="highlighted"{% endif %}>{{ item.text }}</li>
-			{% endfor %}
-
-			{% if feature.call_to_action %}
-				<li class="features-cta" style="margin-top: auto;"><div class="features-cta start"><a href="{{ feature.call_to_action.link }}">{{ feature.call_to_action.text }} &rarr;</a></div></li>
-			{% endif %}
-		</ul>
+	<ul class="feature" style="display: flex; flex-direction: column">
```
Comment (leecalcote) [inline collections/_pages/features.html:89]: Is there any change here other than line wrapping?

Comment (soham4abc) [reply collections/_pages/features.html:89]: No, Only line wrapping.

File: collections/_pages/features.html (hunk 1 above)
Comment (soham4abc) [reply collections/_pages/features.html:29]: okay pushing changes

File: collections/_pages/features.html (hunk 2 above)
Comment (leecalcote) [reply collections/_pages/features.html:89]: Gotcha. 👍

File: collections/_pages/features.html [hunk 3]
```diff
@@ -16,30 +16,32 @@
...
-      - text: Operatore with best practices
+      - text: Operate with confidence
         highlight: true
       - text: Built-in industry patterns and known best practices
         highlight: false
       - text: Apply custom configuration to your service mesh
         highlight: false
       - text: Advanced reporting
+      - highlight: false
+      - text: Management of task/operation lifecycle for Operators Actors using CloudEvents specification
```
Comment (leecalcote) [inline collections/_pages/features.html:37]:
```suggestion
      - text: Management events compatible with the CloudEvents specification
```

**Comment by leecalcote** [comment]
Thanks, @soham4abc 👍 

Have a look [here](https://github.com/layer5io/meshery.io/pull/456/checks?check_run_id=2631140866) to sign-off on your commits.

**Comment by soham4abc** [comment]
> Thanks, @soham4abc 👍
> 
> Have a look [here](https://github.com/layer5io/meshery.io/pull/456/checks?check_run_id=2631140866) to sign-off on your commits.

I did, took me some time! Thanks for guiding.

**Comment by soham4abc** [comment]
I guess it's okay now!

**Review by mgfeller — APPROVED** [review]

**Comment by soham4abc** [comment]
reopening after resolving merge conflicts

**Pull request closed without merging**